system/loggerd/encoderd.cc

```diff
@@ -147,14 +148,21 @@ void encoderd_thread() {
   }
 }
 
-int main() {
+int main(int argc, char* argv[]) {
   if (!Hardware::PC()) {
     int ret;
     ret = util::set_realtime_priority(52);
     assert(ret == 0);
     ret = util::set_core_affinity({3});
     assert(ret == 0);
   }
-  encoderd_thread();
-  return 0;
+  if (argc > 1) {
+    std::string arg1(argv[1]);
+    if (arg1 == "--stream") {
+      encoderd_thread(stream_cameras_logged);
+    }
```

Comment:
add an else that shows a good error message. currently will exit silently with a good exit code